app.py

```diff
@@ -0,0 +1,80 @@
+import os
+from datetime import timedelta
+
+from dotenv import load_dotenv
+from flask import Flask, request
+from membrane.client import (
+    blueprint,
+    configure,
+    membrane_current_user,
+    membrane_login_required,
+)
+
+load_dotenv()
+
+# Constants
+ACTIVATE_LOGIN = os.getenv("FLASK_EXAMPLE_ACTIVATE_LOGIN", "").lower() == "true"
```

Comment:
Why not just look at whether MEMBRANE_SERVER is defined or not?

Reply:
Done (check `config.py`, `app_creator` and `main.py`). I will modify the package in membrane-backend to match. `configure_membrane` will have an `active` parameter that the user can set based on `MEMBRANE_SERVER`. `membrane_login_required` will check for auth only if `active` is set to `True`.